fe/fe-core/src/main/java/org/apache/doris/datasource/ExternalTable.java

```diff
@@ -466,4 +466,13 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hashCode(name, db);
     }
+
+    public boolean isTable() {
```

Comment:
```suggestion
    protected boolean isTable() {
```
 And add comment for this interface